### PR TITLE
MAUI 874348 : Updated UG correction in MAUI Scheduler controls.

### PR DESCRIPTION
**Pull request by Shalini-Ashokan**
### Description
Updated UG correction in MAUI Image editor, DataForm and schedular controls

### CODE REVIEW

File: MAUI/ImageEditor/customview.md [hunk 1]
```diff
@@ -210,7 +210,7 @@ Add a custom view on image loading using the [`Imageloaded`](https://help.syncfu
 {% endtabs %}
 
 
-### Add custom view with manual bounds
+## Add custom view with manual bounds
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/customview.md:213]: Do these changes for text and shape files

File: MAUI/DataForm/getting-started.md [hunk 2]
```diff
@@ -435,3 +435,32 @@ RadioGroup
 </td>
 </tr>
 </table>
+
+## Label view customization
+
+Customize the label view of default layout by using the `InitializeDataLabel` method of the [DataFormItemManager](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.DataFormItemManager.html).
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/getting-started.md:441]: Customize the label view of the default layout by using the `InitializeDataLabel` method of the [DataFormItemManager](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.DataFormItemManager.html).

File: MAUI/Scheduler/cell-selection.md [hunk 3]
```diff
@@ -35,7 +35,7 @@ this.Scheduler.CellSelectionView.Stroke = Brush.Green;
 The cell selection background color can be customized by using the `Background` property in the `CellSelectionView`.
 The default value is Brush.Transparent.
```
Comment (karthikraja-arumugam) [inline MAUI/Scheduler/cell-selection.md:36]: The default value is `Brush.Transparent`.

File: MAUI/Scheduler/cell-selection.md [hunk 4]
```diff
@@ -76,7 +76,7 @@ this.Scheduler.CellSelectionView.CornerRadius = 2;
 
 The thickness of the `Stroke` of Scheduler can be customized by using the `StrokeWidth` property in the `CellSelectionView`.
```
Comment (karthikraja-arumugam) [inline MAUI/Scheduler/cell-selection.md:77]: The thickness of the selection view `Stroke` can be customized by using the `StrokeWidth` property in the `CellSelectionView`.

File: MAUI/Scheduler/cell-selection.md [hunk 5]
```diff
@@ -76,7 +76,7 @@ this.Scheduler.CellSelectionView.CornerRadius = 2;
 
 The thickness of the `Stroke` of Scheduler can be customized by using the `StrokeWidth` property in the `CellSelectionView`.  
 
-Note: The `StrokeWidth` is only applicable for `Stroke` property.
+N> The thickness of the selection view `Stroke` can be customized by using the `StrokeWidth` property in the `CellSelectionView`.
```
Comment (karthikraja-arumugam) [inline MAUI/Scheduler/cell-selection.md:79]: The `StrokeWidth` is only applicable for the `Stroke` property.

File: MAUI/ImageEditor/customview.md [hunk 6]
```diff
@@ -132,7 +132,7 @@ N> This is common for Shape, Text and CustomView annotations.
 
 ## Annotation selected event
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/customview.md:133]: Change file name, custom-view update in toc file also

Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/customview.md]: Changed file name and updated in toc file

File: MAUI/Scheduler/cell-selection.md (hunk 5 above)
Comment (Shalini-Ashokan) [reply MAUI/Scheduler/cell-selection.md]: Updated

File: MAUI/Scheduler/cell-selection.md (hunk 4 above)
Comment (Shalini-Ashokan) [reply MAUI/Scheduler/cell-selection.md]: Updated

File: MAUI/Scheduler/cell-selection.md (hunk 3 above)
Comment (Shalini-Ashokan) [reply MAUI/Scheduler/cell-selection.md]: Updated

File: MAUI/DataForm/getting-started.md (hunk 2 above)
Comment (Shalini-Ashokan) [reply MAUI/DataForm/getting-started.md]: Updated

File: MAUI/ImageEditor/customview.md (hunk 1 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/customview.md:213]: Updated

**Review by karthikraja-arumugam — APPROVED** [review]

**Pull request merged**